Utilities/Thread.cpp

```diff
@@ -1817,17 +1809,7 @@ static void signal_handler(int sig, siginfo_t* info, void* uct) noexcept
 
 	fmt::append(msg, "Thread id = %s.\n", std::this_thread::get_id());
 
-	sys_log.fatal("\n%s", msg);
-	std::fprintf(stderr, "%s\n", msg.c_str());
-
-	if (IsDebuggerPresent())
-	{
-		// Convert to SIGTRAP
-		raise(SIGTRAP);
-		return;
-	}
-
-	report_fatal_error(msg);
+	thread_ctrl::emergency_exit(msg);
```

Comment:
I'd recommend to be more careful with non-windows path. `raise(SIGTRAP)` was more tested than plain int3.

Reply:
Okay, remained unchanged for now.